stock_quant_manual_assign/wizard/assign_manual_quants_view.xml

```diff
@@ -12,9 +12,9 @@
                         <field name="package_id" groups="stock.group_tracking_lot" />
                         <field name="owner_id" groups="stock.group_tracking_owner" />
                         <field name="location_id" />
-                        <field name="on_hand" force_save="1" />
-                        <field name="reserved" force_save="1" />
-                        <field name="selected" widget="boolean_toggle" />
```

Comment:
Why removing the widget?

Reply:
With the widget, there is an error message "Create/update: a mandatory field is not set". I tested new changes without removing the widget because I was curious too.

Reply:
And then why remove the `force_save` switch? Anyway, the bug seems to be in the widget itself, or in the form missing a field. This is only a workaround removing an interesting feature.

Reply:
I did not test without force_save. 

But as far as I saw, the toggle and force_save never existed in older versions. It was added in version 16.0 and never really worked with this module. I don't know why.

Reply:
I don't think `force_save` leads to issue. As far as I checked, I don't see the process that remove `assign_wizard` field when we change boolean field. So, I think the bug seems to be in the widget.